app/views/posts/_write_post.haml

```diff
@@ -41,6 +43,18 @@
         = f.hidden_field :icon_id, id: 'reply_icon_id'
         = submit_tag (method == :put ? 'Save' : "Post"), class: 'button', id: 'submit_button', disable_with: 'Saving...'
         = submit_tag "Preview", class: 'button', id: 'preview_button', name: 'button_preview'
+      %br
+      #markup-allowed-box{style: 'display:none'}
```

Comment:
Possibly? I'm not sure what we'd actually change the UI *to*, because people don't give much input on this, unfortunately, and I don't predict we'll get around to changing it anytime soon, so I think I'd be inclined against, but not particularly strongly so.